nwx_build_environment/buildenv.sh

```diff
@@ -0,0 +1,10 @@
+docker build -t nwx_buildenv -f nwx_buildenv.dockerfile .
+
+deps=("cereal" "madness" "tiledarray" "gauxc" "libfort" "catch2" "spdlog")
+for dep in "${deps[@]}"
+do
+    docker build -t nwx_buildenv -f add_${dep}.dockerfile --build-arg COMPILER=gcc-9 .
+    docker build -t nwx_buildenv -f add_${dep}.dockerfile --build-arg COMPILER=gcc-11 .
+    docker build -t nwx_buildenv -f add_${dep}.dockerfile --build-arg COMPILER=clang-11 .
```

Comment:
Building the package with multiple compilers and then put the libraries into the same image?

Reply:
Yes, we won't the dependencies built with the same compiler that we're building the library with. Each of those is installed in a different directory depending on the compiler, e.g. `/nwx_dependencies/{compiler}`.